libs/cli/src/main/java/org/elasticsearch/cli/Terminal.java

```diff
@@ -151,6 +151,10 @@ public final boolean promptYesNo(String prompt, boolean defaultYes) {
         }
     }
 
+    public void flush() {
+        this.getErrorWriter().flush();
```

Comment:
It's puzzling to me that flush only flushes the error stream but not the output stream?